R/exists.R

```diff
@@ -33,6 +36,20 @@
 #' docdb_exists(src, "mtcars")
 #' docdb_exists(src, "asdfasf")
 #' }
+#' 
+#' # MongoDB
+#' (src <- src_mongo())
+#' docdb_create(src, "mtcars", mtcars)
+#' docdb_exists(src, "mtcars")
+#' docdb_exists(src, "asdfasf")
+#' }
```

Comment:
mismatched brace here, and another below the sqlite example. 

Also, please remove the mongodb example and the changed mongodb exists method so that this PR is isolated to only sqlite changes